server/src/features/CodeLensRequestHandler.ts

```diff
@@ -0,0 +1,36 @@
+import { CodeLens, CodeLensParams } from 'vscode-languageserver';
+
+import { YmlDefinitionProvider } from '../definitions';
+import { YmlFunction } from '../yml-objects';
+
+/**
+ * Create a request handler for the event `foldingRanges`.
+ * The handler, when receiving a `FoldingRangeParams` instance will find
+ * in the provided YmlDefinitionProvider every YML Objects
+ * or YML functions of the text document the request is about
+ * and build a list of `FoldingRange` that will be used by client-side to fold/unfold the code.
+ *
+ * @param definitionsProvider an YmlDefinitionProvider instance to get known YML objects and functions from.
+ *
+ * @returns a request handler for the event `foldingRanges`.
```

Comment:
Wait wut? Why does this block mention folding ranges lol? Copypasted? Or does _that_, too, handle folding ranges?

Reply:
Oops. Wrong copypaste.